changelog/fragments/1764777232-ingest-GZIP-logs-is-GA-and-enabled-by-default.yaml

```diff
@@ -0,0 +1,32 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: feature
+
+# Change summary; a 80ish characters long description of the change.
+summary: "Update GZIP support to GA and enable it by default on filestream"
+
+# Long description; in case the summary is not enough to describe the change
+# this field accommodate a description without length limits.
+# NOTE: This field will be rendered only for breaking-change and known-issue kinds at the moment.
+description: |
+    Ingesting GZIP-compressed files is now GA and enabled by default on filestream, deprecating `gzip_experimental` and introducing `gzip_disabled`. Refer to the [documentation](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-filestream#reading-gzip-files) for more details.
```

Comment:
We don't need a `gzip_disabled`, doesn't `exclude_files: [".gz$"]` do the exact same thing and is actively used basically everywhere already?